lib/python/picongpu/plugins/data/compare.py

```diff
@@ -0,0 +1,222 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sophie Rudat
+License: GPLv3+
+"""
+from picongpu.plugins.emittance import EmittanceData
+from picongpu.plugins.energy_histogram import EnergyHistogramData
+
+import numpy as np
+import os
+import collections
+import json
+
+
+class CompareData(object):
+    """
+    Data Reader for the Energy Histogram Plugin.
+    """
+
+    def __init__(self, run_directory):
+        """
+        Parameters
+        ----------
+        simulation_directory : string
+            path to the run directory of PIConGPU
+            (the path before ``simOutput/``)
+        """
+        if run_directory is None:
+            raise ValueError('The run_directory parameter can not be None!')
+
+        self.run_directory = run_directory
+        self.json_suffix = "scan_ranges.json"
```

Comment:
this should probably be the `self.data_file_suffix` parameter to be consistent with the other readers.

Reply:
@kossag14 please work-in this comment as well